cloudbuild.yaml

```diff
@@ -57,6 +57,7 @@ steps:
   - 'DPU_IMAGE_NAME=${_DPU_IMAGE_NAME}'
   - 'CLOUD_BUILD_PAT_TRIGGER_ID=${_CLOUD_BUILD_PAT_TRIGGER_ID}'
   - 'SECRET_KEY=${_SECRET_KEY}'
+  - 'NEXT_PUBLIC_SECRET_KEY=${_SECRET_KEY}'
```

Comment:
Remove the `SECRET_KEY` environment variable in the previous line.

Also, "SECRET_KEY" is not a descriptive name. Please use a more descriptive name.

Reply:
maybe something like `NEXT_PUBLIC_PAT_ENCRYPTION_SECRET_KEY`

Reply:
ohh no, I need the `SECRET_KEY` env var on the server side too. I was not sure if `NEXT_PUBLIC` prefix with the var name can be used on server side too hence used it it twice for server-side as well as client-side.

Reply:
It can be used everywhere.